src/bitcoin-cli.cpp

```diff
@@ -51,7 +50,6 @@
 using CliClock = std::chrono::system_clock;
 
 const std::function<std::string(const char*)> G_TRANSLATION_FUN = nullptr;
-UrlDecodeFn* const URL_DECODE = urlDecode;
```

Comment:
Was this doing anything, or did we forget to remove it earlier?

Reply:
Not sure I understand your question, do you mean this line in particular or the whole hooking thing? This line was added the with the rest of hooking code in #18504 and I don't anything was changed about that until now, so I don't know where we might have forgotten to remove it.